logger.go

```diff
@@ -1,12 +1,36 @@
 package negroni
 
 import (
+	"bytes"
+
 	"log"
 	"net/http"
 	"os"
+	"sync"
+	"text/template"
 	"time"
 )
 
+// Log is the structure
+// passed to the template.
+type Log struct {
+	StartTime string
+	Status    int
+	Duration  time.Duration
+	Hostname  string
+	Method    string
+	Path      string
+}
+
+// DefaultFormat is the format
+// logged used by the default Logger instance.
+var DefaultFormat = "{{.StartTime}} | {{.Status}} | \t {{.Duration}} | {{.Hostname}} | {{.Method}} {{.Path}} \n"
+
+// DefaultDateFormat is the
+// format used for date by the
+// default Logger instance.
+var DefaultDateFormat = time.RFC3339
+
```

Comment:
I would like this struct type and the related variables to be effectively namespaced to the logger middleware.  Rather than introducing another package level, could you rename these to the following?

- `type Log struct` ➡️ `type LoggerEntry struct`
- `var DefaultFormat` ➡️ `const LoggerDefaultFormat`
- `var DefaultDateFormat` ➡️ `const LoggerDefaultDateFormat`